apps/bfd-model/bfd-model-codebook-library/src/main/java/gov/cms/bfd/model/codebook/model/Value.java

```diff
@@ -72,15 +51,6 @@ public String getDescription() {
     return description;
   }
 
-  /**
-   * Sets the description.
-   *
-   * @param description the new value for {@link #getDescription()}
-   */
-  public void setDescription(String description) {
-    this.description = description;
-  }
-
   /** {@inheritDoc} */
   @Override
   public String toString() {
```

Comment:
Not sure if we're using this toString to validate something and need it in this particular format, but lombok also has a toString annotation.

Reply:
Sounds good, I modified this file to use @ToString